cypress/integration/cucumber/step_definitions/oclsubscriptionmodule/index.js

```diff
@@ -0,0 +1,82 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+const userApiToken = 'e6415c890158e06c36f7f52eca335bdf7204d50f';
+const sUrl = 'https://api.staging.openconceptlab.org/users/openmrs/collections/hdz/1/';
+
+Given('the user goes to the Dictionary Manager App', () => {
+  cy.on('uncaught:exception', (err, runnable) => {
+    console.log(err);
+    return false;
+  });
+  cy.visit('https://openmrs.staging.openconceptlab.org/login');
```

Comment:
Before the user goes to openmrs, i think the user should first take in functionalities of open concept lab , Like that of adding concept dictionary, check the subscription URL.

Reply:
That is already covered in the Dictionary Manager, actually even copying the subscription URL is already done but I repeated it since it's the ideal user workflow.

Reply:
Ohh this sounds like good deal, how will the automation pick this from dictionary manager, is there a connection for it